src/components/features/decoupled-components/solution/solution.tsx

```diff
@@ -0,0 +1,59 @@
+import {
+  ComponentSolutionSectionFragment
+} from '@src/data/contentful/solution/__generated/solution.generated';
+import { Teaser } from '@src/components/features/decoupled-components/teaser/teaser';
+import { Title } from '@src/components/features/decoupled-components/title/title';
+import {TitleProps} from "@src/components/features/decoupled-components/title/title.typings";
+
+
+/* export interface Teaser {
```

Comment:
Typings should be kept together with the component they belong to, please move this to where it belongs and delete this comment, thanks :)